src/common/utils/currencyConvert.ts

```diff
@@ -27,7 +23,7 @@ export const getFiatValue = async ({
   rate: number;
   currency: CURRENCIES;
 }) => {
-  const fiatValue = await satoshisToFiat({
+  const fiatValue = satoshisToFiat({
```

Comment:
Oh nice.

This allows an even more simplification:

```TS

export const getFiatValue = ({  // remove async
  amount,
  rate,
  currency,
}: {
  amount: number | string;
  rate: number;
  currency: CURRENCIES;
}) => {
  const fiatValue = Number(amount) * rate); // delete func satoshisToFiat above

  return fiatValue.toLocaleString("en", {
    style: "currency",
    currency,
  });
};
```


And remove the `await` from all places where `getFiatValue`  is used